requests-for-comments/2024-08-14-youtuber.md

```diff
@@ -0,0 +1,53 @@
+
+**Why not another (writing) content person?** We need to fill for James. There is always more newsletters, docs, tutorials, and blogs to do, but I don’t feel it is enough to add another content person (beyond filling in for James).
+
+## We shouldn’t wait for the community manager
+
+Review comments:
+
+> We should wait for the community manager
+
+I do see where you are coming from on a community manager front, this will likely overlap with what their work is, but I disagree on it being the reason we should not hire someone here reasons.
+
+1. The job listing for community manager went out June 7th-ish. This shows it takes a long time to hire for a role. It will take them more time to decide to join, onboard, and get used to the PostHog culture. It is a big unknown, both in terms of what they want to do and when it will happen. 
+
+2. Unless I am misunderstanding, nowhere in the community manager role does it say they will be creating the content or deciding what content we will be creating. I think content creation sits squarely in the marketing camp even though we will of course be collaborating and working with the community manager.
+
+3. I do think the YouTuber will have a better idea of what content will work on YouTube. We already have some sense of this, but don’t have the bandwidth to execute. For example, more content like: [demo](https://www.youtube.com/watch?v=2jQco8hEvTI), [Clickhouse vs Postgres](https://www.youtube.com/watch?v=iLXXoDaFoxs), [event tracking guide](https://www.youtube.com/watch?v=LIJ_TuyMq74), [modern data stack sucks](https://www.youtube.com/watch?v=2N2cvCmv4us). The community manager will also be scattered with other initiatives and not dedicate themselves to YouTube like I am suggesting. 
+
+4. I still see influencer work sitting entirely with the community manager. It makes it EASIER for them to do their job by having a YouTuber to collaborate with. It brings us to the attention of more YouTubers. 
+
+I’m perfectly fine with this being a “no, we shouldn’t hire a YouTuber” but I don’t think “we should let the community manager decide” is the right answer. Other ones that I think could be better reasons:
+
+- We don’t want to spend more on content, would rather focus on paid ads and other growth marketing.
+- We really don’t think YouTube would work even though it is driving a significant amount of signup mentions.
+- We want to hire this role, but we have too much on our plate hiring a content marketer and community manager. We should wait until they are hired and hire someone 1-3 months later. Not because they will decide for us, but because we don’t have time.
+- Something else 🤷
+
+### Next steps:
+
+- Charles + Andy + James: Yes/No
+- Create job post
+- Ian reaches out to influencers we already work with
```

Comment:
Flagging Eric Roby, who we've worked previously for the tutorials. He's a software engineer too. Here's his own channel - https://www.youtube.com/@codingwithroby